packages/wpcom-checkout/test/checkout-line-items.tsx

```diff
@@ -0,0 +1,75 @@
+import { getEmptyResponseCartProduct } from '@automattic/shopping-cart';
+import { shallow } from 'enzyme';
```

Comment:
FWIW I think there's a project currently replacing `enzyme` with `testing-library`, eg:

- https://github.com/Automattic/wp-calypso/pull/65517
- https://github.com/Automattic/wp-calypso/pull/65234
- https://github.com/Automattic/wp-calypso/pull/65140
- https://github.com/Automattic/wp-calypso/pull/65019
- etc.

We might not want to introduce another test that needs to be migrated. Could this use `testing-library` instead?

Reply:
Thanks for the review @sirbrillig! And thanks for sharing examples about migrating to `testing-library`. (TIL!)
I have updated the PR and migrated the tests to `testing-library`.